bedrock/firefox/templates/firefox/nightly_firstrun.html

```diff
@@ -4,9 +4,7 @@
 
 {# firstrun for nightlies #}
 
-{% extends "/firefox/base-resp.html" %}
-
-{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+{% extends "/firefox/whatsnew-base-resp.html" %}
```

Comment:
Also seems confusing that firstrun templates are now extending `whatsnew-base-resp.html`. Should probably find a better name?